plugins/LilyPad/Config.cpp

```diff
@@ -933,9 +945,8 @@ int LoadSettings(int force, wchar_t *file)
         }
     }
 
-    for (int i = 0; i < sizeof(BoolOptionsInfo) / sizeof(BoolOptionsInfo[0]); i++) {
+    for (int i = 0; i < sizeof(BoolOptionsInfo) / sizeof(BoolOptionsInfo[0]); i++)
         config.bools[i] = GetPrivateProfileBool(L"General Settings", BoolOptionsInfo[i].name, BoolOptionsInfo[i].defaultValue, file);
-    }
```

Comment:
While I personally prefer not using braces for single statement if/else/for/while, this is kind of a pointless change since you're not changing anything else and should be avoided in large commits.